Automattic/woocommerce-payments
# Update `@woocommerce/experimental` package to v2.1.0

#### Changes proposed in this Pull Request

This PR updates the `@woocommerce/experimental` package to version `2.1.0`.
Version [2.1.0 adds](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/experimental/CHANGELOG.md#210) a [prop rename for the TaskItem](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/experimental/src/experimental-list/task-item/index.tsx#L46) component. That prop rename was also applied for the `TaskItem` uses in this repo.










#### Testing instructions





* Just check out this branch and do some smoke tests on the `Overview` page and on the `Things to do` task list to confirm that nothing is broken.

-------------------
